WalletWasabi.Gui/Controls/WalletExplorer/PhaseState.cs

```diff
@@ -0,0 +1,17 @@
+﻿using WalletWasabi.CoinJoin.Common.Models;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
```

Comment:
Move this file into WalletWasabi.Gui.Models.

Reply:
done